.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Expected behavior
```

Comment:
We could use this format for it this step. Make sure people describe the behavior in steps

- When I perform a backup
- I expect it to backup